lib/pf/UnifiedApi/Controller/Config/FilterEngines/DHCPFilters.pm

```diff
@@ -0,0 +1,59 @@
+package pf::UnifiedApi::Controller::Config::FilterEngines::DHCPFilters;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Config::FilterEngines::DHCPFilters - 
+
+=cut
+
+=head1 DESCRIPTION
+
+Configure dhcp filters
+
+=cut
+
+use strict;
+use warnings;
+
+use Mojo::Base qw(pf::UnifiedApi::Controller::Config);
+
+has 'config_store_class' => 'pf::ConfigStore::DhcpFilters';
+has 'form_class' => 'pfappserver::Form::Config::FilterEngines::DHCPFilter';
+has 'primary_key' => 'dhcp_filter_id';
+
+use pf::ConfigStore::DhcpFilters;
+use pfappserver::Form::Config::FilterEngines::DHCPFilter;
+
+sub is_sortable {
```

Comment:
This code is being repeated in a lot of controllers, a role would be better (or a util function function you reuse)